nitransforms/io/itk.py

```diff
@@ -89,33 +155,74 @@ def __init__(self,
                  check=True):
         """Initialize with (optionally) a list of transforms."""
         super().__init__(binaryblock, endianness, check)
-        self._xforms = []
-        for i, mat in enumerate(xforms or []):
-            xfm = ITKLinearTransform()
-            xfm['parameters'] = mat
-            xfm['index'] = i + 1
-            self._xforms.append(xfm)
+        self.xforms = [ITKLinearTransform(parameters=mat)
+                       for mat in xforms or []]
+
+    @property
+    def xforms(self):
+        """Get the list of internal ITKLinearTransforms."""
+        return self._xforms
+
+    @xforms.setter
+    def xforms(self, value):
+        self._xforms = value
+
+        # Update indexes
+        for i, val in enumerate(self._xforms):
+            val['index'] = i
 
     def __getitem__(self, idx):
         """Allow dictionary access to the transforms."""
         if idx == 'xforms':
             return self._xforms
         if idx == 'nxforms':
             return len(self._xforms)
-        return super().__getitem__(idx)
+        raise KeyError(idx)
+
+    def to_filename(self, filename):
+        """Store this transform to a file with the appropriate format."""
+        if str(filename).endswith('.mat'):
+            raise TransformFileError('Please use the ITK\'s new .h5 format.')
+
+        with open(str(filename), 'w') as f:
+            f.write(self.to_string())
+
+    def to_ras(self):
+        """Return a nitransforms' internal RAS matrix."""
+        return np.stack([xfm.to_ras() for xfm in self._xforms])
```

Comment:
let's access through the property
```suggestion
        return np.stack([xfm.to_ras() for xfm in self.xforms])
```